src/components/SubMenu.vue

```diff
@@ -1,17 +1,25 @@
 <template>
-  <q-menu auto-close>
+  <q-menu ref="qMenu" :anchor="anchor" :self="self">
     <q-list style="min-width: 100px">
       <q-item
         clickable
         v-for="menu in menuItem.attributes.children.data"
         :key="menu.id"
+        @click="close(menu)"
       >
-        <q-item-section>{{ getLocalizedTitle(menu) }} </q-item-section>
-        <SubMenu
-          v-if="menu.attributes.children.data.length > 0"
-          :menuItem="menu"
-          :level="level + 1"
-        />
+        <q-item-section>{{ getLocalizedTitle(menu) }}</q-item-section>
+        <template v-if="menu.attributes.children.data.length > 0">
+          <q-item-section side>
+            <q-icon name="keyboard_arrow_right" />
+          </q-item-section>
+          <SubMenu
+            :menuItem="menu"
+            :level="level + 1"
+            anchor="top end"
+            self="top start"
+            @close="close"
```

Comment:
When closing a submenu, we should propagate the close event to all menus in the hierarchy.